src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -123,12 +176,84 @@ export class SentryAppExternalIssueForm extends React.Component<Props> {
     }
   }
 
+  handleFieldChange = async (id: string, _value: FieldValue) => {
+    const config = this.state;
```

Comment:
I think you should refactor this a bit to use the `setState` callback method since there are some async things happening, `config` can be changed by the time you call setState

Reply:
@billyvg What do you think should go in the callback for `setState`?

Reply:
Sorry not callback, but the other form of `setState`: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous